lib/footprint-translators/get-jlc-package-from-footprinter-string.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Transforms a generic "footprinter string" into a JLC-compatible package name.
+ * e.g. "0603cap" -> "0603"
```

Comment:
this isn't a valid footprinter string

```suggestion
 * e.g. "cap0603" -> "0603"
```